core/src/main/scala/org/apache/spark/storage/DiskBlockObjectWriter.scala

```diff
@@ -272,6 +279,25 @@ private[spark] class DiskBlockObjectWriter(
     file
   }
 
+  /**
+   * Reverts write metrics that haven't been committed yet and delete the file held
```

Comment:
Please fix: Reverts write metrics ~~that haven't been committed yet~~ and delete the file held

Reply:
OK

Reply:
c98ff1d fix this